bindings/redis/redis.go

```diff
@@ -66,21 +64,41 @@ func (r *Redis) Ping() error {
 }
 
 func (r *Redis) Operations() []bindings.OperationKind {
-	return []bindings.OperationKind{bindings.CreateOperation}
+	return []bindings.OperationKind{
+		bindings.CreateOperation,
+		bindings.DeleteOperation,
+		bindings.GetOperation,
+	}
 }
 
 func (r *Redis) Invoke(ctx context.Context, req *bindings.InvokeRequest) (*bindings.InvokeResponse, error) {
-	if val, ok := req.Metadata["key"]; ok && val != "" {
-		key := val
-		_, err := r.client.Do(ctx, "SET", key, req.Data).Result()
-		if err != nil {
-			return nil, err
+	if key, ok := req.Metadata["key"]; ok && key != "" {
+		switch req.Operation {
+		case bindings.DeleteOperation:
+			err := r.client.Del(ctx, key).Err()
+			if err != nil {
+				return nil, err
+			}
+		case bindings.GetOperation:
+			data, err := r.client.Get(ctx, key).Result()
+			if err != nil {
+				return nil, err
+			}
+			rep := &bindings.InvokeResponse{}
+			rep.Data = []byte(data)
+			return rep, nil
+		case bindings.CreateOperation:
+			_, err := r.client.Do(ctx, "SET", key, req.Data).Result()
+			if err != nil {
+				return nil, err
+			}
+		default:
+			return nil, errors.Errorf("invalid operation type: %s. Expected %s, %s, or %s",
```

Comment:
`return nil, fmt.Errorf("invalid operation type: %s", req.Operation)`

Users can get the list of supported operations by invoking the Features method

Reply:
ok, but this will be a bit inconsistent with /bind/mysql.go